pkg/appgw/requestroutingrules_test.go

```diff
@@ -937,4 +937,86 @@ var _ = Describe("Test routing rules generations", func() {
 		})
 
 	})
+
+	Context("test pathType in basic ingress", func() {
+		configBuilder := newConfigBuilderFixture(nil)
+		service := tests.NewServiceFixture(*tests.NewServicePortsFixture()...)
+		ingress := tests.NewIngressTestWithVariousPathTypeFixture(tests.Namespace, "random")
+		rewriteRuleSetName := "custom-response-header"
+		ingress.Annotations[annotations.RewriteRuleSetKey] = rewriteRuleSetName
+
+		cbCtx := &ConfigBuilderContext{
+			IngressList:           []*networking.Ingress{&ingress},
+			ServiceList:           []*v1.Service{service},
+			DefaultAddressPoolID:  to.StringPtr("xx"),
+			DefaultHTTPSettingsID: to.StringPtr("yy"),
+		}
+
+		_, urlPathMaps := configBuilder.getRules(cbCtx)
+		pathRules := *urlPathMaps[0].PathRules
+
+		It("should have 8 paths", func() {
+			Expect(len(pathRules)).To(Equal(8))
+		})
+
+		It("should add * to paths with pathType:prefix", func() {
```

Comment:
how about to check:
1. path: /  with prefix pathType
2. path: /* with exact pathType

Reply:
Path "/" is not supported with exact on AppGW. It throws a validation error when AGIC created a pathRule with path "/" saying that a suffix is needed.

Reply:
So, AGIC needs to treat "/" and pathType:exact as a prefix match.